### PR TITLE
chore(integration tests): add admin collection mutation integration tests

### PR DESCRIPTION
**Pull request by jpetto**
## Goal

add admin collection mutation integration tests

- remove legacy database integration tests
- migrate a couple more collection partner integration tests

## Reference

Tickets:

- https://getpocket.atlassian.net/browse/BACK-1379

### CODE REVIEW

File: src/admin/resolvers/queries/CollectionPartner.integration.ts [hunk 1]
```diff
@@ -166,7 +169,41 @@ describe('queries: CollectionPartner', () => {
...
+      });
+
+      const {
+        data: { getCollectionPartnerAssociation: data },
+      } = await server.executeOperation({
+        query: GET_COLLECTION_PARTNER_ASSOCIATION,
+        variables: { externalId: 'invalid-id' },
+      });
+
+      expect(data).to.be.null;
```
Comment (Herraj) [inline src/admin/resolvers/queries/CollectionPartner.integration.ts:206]: should we add a check for `errors` in the returned response?

Comment (jpetto) [reply src/admin/resolvers/queries/CollectionPartner.integration.ts:206]: you would think! i started this route, but then found out the `errors` array is never populated for this scenario. the `TODO` i added in another file in this PR addresses this issue. i also made a ticket to follow up: https://getpocket.atlassian.net/browse/BACK-1399

Comment (jpetto) [reply src/admin/resolvers/queries/CollectionPartner.integration.ts:206]: fwiw this test is mimicking prior test behavior, so we aren't _losing_ any coverage.

**Review by Herraj — APPROVED** [review]
:shipit:

**Pull request merged**